wagtail/users/templatetags/wagtailusers_tags.py

```diff
@@ -55,6 +56,14 @@ def normalize_permission_label(permission: Permission):
     return label
 
 
+# normalize_permission_label will return "Can view" for Django's standard "Can view X" permission.
+# formatted_permissions.html passes these labels through {% trans %} - since this is a variable
+# within the template it will not be picked up by makemessages, so we define a translation here
+# instead.
+
+VIEW_PERMISSION_LABEL = _("Can view")
```

Comment:
How about, changing the lines here:

https://github.com/wagtail/wagtail/blob/538365fcfc5ea9135663a1b8c2511c499910e419/wagtail/users/templatetags/wagtailusers_tags.py#L45-L46

to something like

```py
if is_default and permission.name.startswith(f"Can {action}"):
    if action == "view":
        return gettext("Can view")
    return gettext("Can %(action)s") % {"action": action.replace('_', ' ')}
```

and leaving the `formatted_permissions.html` template as-is? It's not perfect, as any `action` other than `"view"` won't get translated, but it allows the "Can" part to be translated too. It allows us to avoid having to define this variable too...

Reply:
One possible downside is that this won't allow developers to define their own set of translatable strings e.g. "Can deliver pizza" to then be translated in the template via `{% trans %}` like we do in this PR.

Perhaps we can keep the `{% trans %}` in the templates, then use [`gettext_noop`](https://docs.djangoproject.com/en/stable/ref/utils/#django.utils.translation.gettext_noop) for the `"Can view"` string, e.g.

```py
if is_default and permission.name.startswith(f"Can {action}"):
    if action == "view":
        return gettext_noop("Can view")
    return f"Can {action.replace('_', ' ')}" 
```

and instruct developers to provide a translatable string for each custom permission as `"Can {action_name_with_underscores_replaced_with_spaces}"`

Reply:
@laymonage Not sure how that improves on what's in the PR right now...?

Reply:
I was brainstorming some ideas, but if we go with my second comment instead of the first one, you're right – it doesn't really change anything from the approach in the PR currently. I just thought moving the translatable "Can view" string to the place where it's needed is more readable than having a separate variable with a comment explaining how it's related to the function.

In any case, it's better to use `gettext_noop` instead of `gettext`/`gettext_lazy` if we don't intend the variable to be translated directly. I'm happy with the current PR's approach + changing `_` to `gettext_noop`.